data/redirects.yml

```diff
@@ -661,7 +661,7 @@
 # Build links (external)
 "https://www.docker.com/build-early-access-program/?utm_campaign=onboard-30-customer-zero&utm_medium=in-product-ad&utm_source=desktop_v4":
   - /go/build-eap/
-"https://build.docker.com/":
+"https://www.docker.com/products/build-cloud/":
```

Comment:
I noticed this page is a 404 currently. Will it be going live tomorrow?

Reply:
yep unless the launch is postponed again